pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/LeastLongTermMessageRate.java

```diff
@@ -60,21 +58,16 @@ private static double getScore(final BrokerData brokerData, final ServiceConfigu
         final TimeAverageBrokerData timeAverageData = brokerData.getTimeAverageData();
         final double maxUsage = brokerData.getLocalData().getMaxResourceUsage();
         if (maxUsage > overloadThreshold) {
-            log.warn("Broker {} is overloaded: message rate={}, max usage={}",
-                    brokerData.getLocalData().getWebServiceUrl(), maxUsage);
+            log.warn("Broker {} is overloaded: max usage={}", brokerData.getLocalData().getWebServiceUrl(), maxUsage);
             return Double.POSITIVE_INFINITY;
         }
-        // 1 / weight is the proportion of load this machine should receive in proportion to a machine with no system
-        // resource burden. This attempts to spread out the load in such a way that machines only become overloaded if
-        // there is too much load for the system to handle (e.g., all machines are at least nearly overloaded).
-        final double weight = 1 / (overloadThreshold - maxUsage);
         final double timeAverageLongTermMessageRate = timeAverageData.getLongTermMsgRateIn()
                 + timeAverageData.getLongTermMsgRateOut();
         final double totalMessageRateEstimate = totalMessageRate + timeAverageLongTermMessageRate;
-        final double score = weight * totalMessageRateEstimate;
-        log.info("Broker {} has long term message rate {}, weight {}, and score of {}",
-                brokerData.getLocalData().getWebServiceUrl(), totalMessageRateEstimate, weight, score);
-        return score;
+
+        log.debug("Broker {} has long term message rate {}",
```

Comment:
typically we wrap all the debug statements within a `if (log.isDebugEnabled()) {....}` to avoid any parameters boxing or `Object[]` vargs allocations.